x-pack/plugins/viz_editor/common/query_types.ts

```diff
@@ -15,11 +15,12 @@ export type SelectOperator =
   | 'date_histogram'
   | 'sum'
   | 'avg'
-  | 'terms';
+  | 'terms'
+  | 'window';
 
 export interface Aliasable {
   operator: SelectOperator;
-  alias?: string;
+  alias: string;
```

Comment:
The way you are currently using this is as an ID. I'm totally fine with this change, but let's stop calling it an alias and require it on all operations.

Reply:
Makes sense, I renamed it to id. It was already required on all of the select operations.